src/explore-education-statistics-admin/src/pages/release/data/components/ReleaseDataUploadsSection.tsx

```diff
@@ -208,12 +218,11 @@ export default function ReleaseDataUploadsSection({
             dataFiles={allDataFiles}
             onSubmit={handleSubmit}
           />
-          {bulkUploadPlan === undefined ||
-          bulkUploadPlan.length === 0 ? null : (
-            <BulkZipUploadModalConfirm
-              bulkUploadPlan={bulkUploadPlan}
-              onConfirm={confirmBulkUploadPlan}
-              onCancel={() => setBulkUploadPlan(undefined)}
+          {uploadResults === undefined || uploadResults.length === 0 ? null : (
```

Comment:
I think this can be simplified to:

```
{uploadResults?.length > 0 && (
        <DataSetUploadModalConfirm
              uploadResults={uploadResults}
              onConfirm={confirmDataSetImport}
              onCancel={() => setUploadResult(undefined)}
         />
)}` 
```

but I could be wrong!